<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackTool</class>
 <widget class="QWidget" name="TrackTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="styleSheet">
       <string notr="true">QWidget { background-color: white; } </string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>70</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>24</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>24</number>
           </attribute>
           <row>
            <property name="text">
             <string>Name</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Type</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Width</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Bus</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Orientation</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Count</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Spacing</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Property</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>Name</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>Type</string>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>Width</string>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>Bus</string>
            </property>
           </item>
           <item row="4" column="0">
            <property name="text">
             <string>Orientation</string>
            </property>
           </item>
           <item row="5" column="0">
            <property name="text">
             <string>Count</string>
            </property>
           </item>
           <item row="6" column="0">
            <property name="text">
             <string>Spacing</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="currentLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Imax:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QComboBox" name="typeComboBox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>12</y>
         <width>109</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox { background-color: white; } </string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Simple</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/doubleangle.png</normaloff>:/pictures/doubleangle.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Angle1</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/oneangleline1.png</normaloff>:/pictures/oneangleline1.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Angle2</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/oneangleline2.png</normaloff>:/pictures/oneangleline2.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right angle 1</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/rightangleline1.png</normaloff>:/pictures/rightangleline1.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right angle 2</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/rightangleline2.png</normaloff>:/pictures/rightangleline2.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Arc 1</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/arc1.png</normaloff>:/pictures/arc1.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Arc 2</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/arc2.png</normaloff>:/pictures/arc2.png</iconset>
        </property>
       </item>
      </widget>
      <widget class="MYUnitsSpinBox" name="widthSpinBox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>53</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="busTypeComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>74</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox { background-color: white; } </string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Horizontal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vertical</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="busCheckBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>320</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QSpinBox" name="busSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>108</x>
         <y>201</y>
         <width>37</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
      </widget>
      <widget class="MYUnitsSpinBox" name="spacingSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>108</x>
         <y>227</y>
         <width>53</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MYUnitsSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>utils/myunitsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../mytypon.qrc"/>
 </resources>
 <connections/>
</ui>
